docs/release.md

```diff
@@ -0,0 +1,21 @@
+# How to Release Epinio
+
+First release epinio/epinio
+* [ ] check CI is green in epinio/epinio
+* [ ] push version tag (vX.X.X)
+* [ ] wait for release, then edit/check release notes on Github
+
+Now relesease both epinio/helm-charts
+* [ ] look for updatecli PR, created by epinio/epinio release
+* [ ] add commit with new `version:` keys
+* [ ] check that `epinioChart:` in chart/epinio-installer/values.yaml is correct
+* [ ] wait for chart CI to finish
+* [ ] update Git submodule in epinio/epinio to point to the tested helm-charts repo
```

Comment:
This is not necessary. We can bump the git submodule at any time. The epinio CI is using chartmuseum to build and use any version currently checked.

Reply:
There was some discussion if this would affect tests in epinio/epinio.